src/grpc/mod.rs

```diff
@@ -178,6 +226,19 @@ impl ChainFollowerRequestService for AyaCardanoRPCServer {
     ) -> Result<Response<EventResponse>, Status> {
         println!("Got a request: {request:?}");
```

Comment:
Can we remove this  `println` statement?

Reply:
I agree with @Eloi :s suggestion. If something malfunctions in production, then we want to see right away where the problem is coming from instead of incurring painful delays by implementing debugging post-mortem

Reply:
Println! replaced with log::debug! everywhere in grpc/mod.rs